spec/services/user_alerts/alert_user_about_new_device_spec.rb

```diff
@@ -91,7 +91,7 @@
         create(:event, user:, event_type: :sign_in_after_2fa, created_at: 16.minutes.ago)
 
         # 2. Exclude events outside the timeframe, e.g. previous sign-in
-        create(:event, user:, event_type: :sign_in_after_2fa, created_at: 4.minutes.ago)
+        # create(:event, user:, event_type: :sign_in_after_2fa, created_at: 4.minutes.ago)
```

Comment:
@aduth should this test still apply? it's failing and I'm not following the logic. An event created 4 minutes ago should still fall in a 15 minute window if I'm not mistaken to how it's working. (see also ln.143 below)

Reply:
There's an implied assumption here that an email would have been sent in response to the previous 2FA, so we don't want to include that again in the new email.